### ISSUE TITLE
Colors are not from colorscheme

### ISSUE THREAD

**Issue by vektah** [oker1/phpunit_codecoverage_display#18]
Great plugin! One little problem in the 0.3.0 release.

The plugin used to use the colors from colors & fonts -> general ->  uncovered line / partial line coverage / full line coverage. Now it defaults to bright red and green (probably FF0000 and 00FF00). 

This is unreadable on dark backgrounds:
![crazy colors](https://f.cloud.github.com/assets/2247982/1241887/02c21f2e-2a2a-11e3-9850-07fc13621b1a.png)

**Comment by WebnetMobile** [comment]
@oker1 can you please look into this in 2015? These bright colors are really hurting my eyes and I need to disable your, otherwise very useful, plugin to be able to work with my code ;( I'd like to have it always on, with my, dimmed colors of choice. Thanks

**Comment by aimfeld** [comment]
@oker1 Great plugin! +1 for using the IDE's color scheme, I have a hard time reading the text with the theme enabled (Darcula)

**Comment by markushausammann** [comment]
+1 :+1:

**Comment by rscafi** [comment]
:+1:

**Comment by Rackberg** [comment]
Whats the status here? I'm facing the same problem using the 0.5.0 release with phpstorm 2016.1

**Comment by nubeiro** [comment]
+1

**Comment by emaas** [comment]
Yes, i'm facing the same issue and would love to see this solved.
I'm using 2016.1.2 on Windows 10
For the rest, it's an amazingly usefull plugin.

**Comment by tarlepp** [comment]
+1

**Comment by therpr** [comment]
It works fine if you set the color in (colors)->Line coverage->Full, you then have to rerun your tests with coverage.
![zrzut ekranu 2016-08-10 o 14 56 41](https://cloud.githubusercontent.com/assets/8452137/17554342/b70a1564-5f0a-11e6-9b12-b3eaf180d3ac.png)

**Comment by REBELinBLUE** [comment]
I have the colours set for Full (484848), Partial (80805A) and Uncovered (715353) but they still show as bright green and red

<edit>Ah, I had them set in foreground not background

**Comment by fracz** [comment]
+1 for this.

Although this plugin works, it's unusable because of the hardcoded colors.

**Comment by kgkg** [comment]
Any chance of addressing this issue, anyone?
